locale/in/LC_MESSAGES/sphinx.po

```diff
@@ -0,0 +1,86 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) 2021 Fortran programming language community
+# This file is distributed under the same license as the fpm package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, 2023.
+#
+#, fuzzy
+msgid ""
+msgstr ""
+"Project-Id-Version: fpm \n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2023-03-10 00:47+0530\n"
+"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
+"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
+"Language: in\n"
+"Language-Team: in <LL@li.org>\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=utf-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Generated-By: Babel 2.12.1\n"
+
+#: ../../../../../../../home/kritika_rag/miniconda3/envs/sphinx/lib/python3.9/site-packages/ablog/templates/archives.html:3
```

Comment:
This looks to me some local path in your system. I think this needs to be updated and all it's other instances.

Reply:
Hii yes I'm so sorry for that. I've made the updates can you review it again.

Reply:
@arteevraina Hi can you please check if it's correct now or not?